// Try Catch el codigo sigue funcionando pero vas a tener un mensaje de error que te 
// va a decir donde se encuentra el error

const numero1 = 20;
const numero3 = 50;

try { // quiere decir intenta hacer lo siguiente
    console.log(numero1);
    console.log(numero2);
    console.log(numero3); 
    
} catch (error) { // en caso que hay un error dime cual es
    console.log(error);
}




